morango/sync/syncsession.py

```diff
@@ -634,8 +639,11 @@ def initialize(self, sync_filter):
             transfer_session=self.current_transfer_session
         )
 
-        # create transfer session on server side and proceed to queuing
+        # backwards compatibility for the queuing signal as it included both serialization
```

Comment:
Since Kolibri is the only (known) consumer of Morango, we'd probably be fine just updating it on the Kolibri side, too, and not worrying about local-only API changes (which I think these signals may be); just need to worry about things that would impact one version of Kolibri talking to another.

(fine to leave like this, though -- but thought it worth noting in case it simplifies)